doc/user-guide/user_guide.md

```diff
@@ -1,71 +1,41 @@
-[Big Query][big-query-dialect-doc]  |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[DB2][db2-dialect-doc]              |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Exasol][exasol-dialect-doc]        |  [Exasol Virtual Schema][exasol-virtual-schema-repository] | [Latest release][exasol-virtual-schema-releases] |                
-[Hive][hive-dialect-doc]            |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Impala][impala-dialect-doc]        |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[MySQL][mysql-dialect-doc]          |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Oracle][oracle-dialect-doc]        |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[PostgreSQL][postgresql-dialect-doc]|  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Redshift][redshift-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[SAP HANA][sap-hana-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[SQL Server][sql-server-dialect-doc]|  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Sybase ASE][sybase-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-[Teradata][teradata-dialect-doc]    |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |          
-Generic                             |  [Virtual Schemas][virtual-schemas-repository]             | [Latest release][virtual-schemas-releases]       |
-
-
-# JDBC Adapter for Virtual Schemas
-
-## Overview
 
 The JDBC adapter for virtual schemas allows you to connect to JDBC data sources like Hive, Oracle, Teradata, Exasol or any other data source supporting JDBC.
 It uses the well proven ```IMPORT FROM JDBC``` Exasol statement behind the scenes to obtain the requested data, when running a query on a virtual table. 
 The JDBC adapter also serves as the reference adapter for the Exasol virtual schema framework.
 
-You can find supported dialects in [the list of supported dialects](#list-of-supported-dialects).
+You can find supported dialects in [the list of supported dialects](dialects.md).
```

Comment:
Make sure that this link works. My browser shows it as a broken link. Check also other linked you on the page and also links to the sections you removed in other parts of documentation

Reply:
I just checkt it agian. For me it works.